packages/wallets/src/components/WalletImport.tsx

```diff
@@ -9,8 +9,26 @@ import {
 import { useForm, useFieldArray } from 'react-hook-form';
 import { useAddKeyMutation, useLogInMutation } from '@chia/api-react';
 import { useNavigate } from 'react-router';
-import { Autocomplete, ButtonLoading, Form, Flex, Logo, useShowError, useTrans } from '@chia/core';
+import {
+  AlertDialog,
+  Autocomplete,
+  Button,
+  ButtonLoading,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Form,
+  Flex,
+  Logo,
+  TextField,
+  useOpenDialog,
+  useShowError,
+  useTrans
+} from '@chia/core';
```

Comment:
The following modules are not used:  
`Dialog`, `DialogActions`, `DialogContent`, `DialogContentText`, `DialogTitle`, `TextField`, `useShowError`.